packages/web-components/examples/autoalign/cdn.html

```diff
@@ -11,10 +11,6 @@
   <head>
     <title>@carbon/web-components autoalign example</title>
     <meta charset="UTF-8" />
-    <link
-      rel="stylesheet"
-      href="https://1.www.s81c.com/common/carbon-for-ibm-dotcom/tag/v1/latest/plex.css"
-    />
```

Comment:
when testing the CDN.html files in a stackblitz we still need the Plex CDN to get the font to come through, but for the index.html files it is not needed

Reply:
unless maybe adding the plex CDN is not necessary for the examples?

Reply:
Good catch! Looks like I got carried away removing stuff.